docs/CompilerPerformance.md

```diff
@@ -559,30 +559,6 @@ compilers on hand while you're working.
        0.0001 (100.0%)   0.0001 (100.0%)   0.0001 (100.0%)   0.0490 (100.0%)  Total
     ```
 
-  - `-Xfrontend -debug-time-compilation`: asks each frontend to print out timers
-    for each phase of its execution. Its output (per-frontend) looks like this:
-
-    ```
-    ===-------------------------------------------------------------------------===
-                                 Swift compilation
-    ===-------------------------------------------------------------------------===
-    Total Execution Time: 0.0876 seconds (0.0877 wall clock)
-
-     ---User Time---   --System Time--   --User+System--   ---Wall Time---  --- Name ---
-     0.0241 ( 53.9%)   0.0394 ( 92.0%)   0.0635 ( 72.5%)   0.0635 ( 72.5%)  Import resolution
-     0.0170 ( 38.0%)   0.0025 (  5.8%)   0.0195 ( 22.3%)   0.0195 ( 22.2%)  Type checking / Semantic analysis
-     0.0013 (  3.0%)   0.0004 (  0.8%)   0.0017 (  1.9%)   0.0017 (  1.9%)  LLVM output
-     0.0010 (  2.3%)   0.0003 (  0.7%)   0.0013 (  1.5%)   0.0013 (  1.5%)  SILGen
-     0.0006 (  1.4%)   0.0002 (  0.4%)   0.0008 (  0.9%)   0.0008 (  0.9%)  IRGen
-     0.0004 (  0.8%)   0.0000 (  0.1%)   0.0004 (  0.5%)   0.0004 (  0.5%)  SIL optimization
-     0.0002 (  0.5%)   0.0001 (  0.1%)   0.0003 (  0.3%)   0.0003 (  0.3%)  LLVM optimization
-     0.0001 (  0.1%)   0.0000 (  0.1%)   0.0001 (  0.1%)   0.0001 (  0.1%)  Parsing
-     0.0000 (  0.0%)   0.0000 (  0.0%)   0.0000 (  0.0%)   0.0000 (  0.0%)  SIL verification (pre-optimization)
-     0.0000 (  0.0%)   0.0000 (  0.0%)   0.0000 (  0.0%)   0.0000 (  0.0%)  SIL verification (post-optimization)
-     0.0000 (  0.0%)   0.0000 (  0.0%)   0.0000 (  0.0%)   0.0000 (  0.0%)  AST verification
-     0.0448 (100.0%)   0.0428 (100.0%)   0.0876 (100.0%)   0.0877 (100.0%)  Total
-    ```
-
```

Comment:
I'm thinking that what was nice about this was that it was an easy way to get per-frontend-per-phase info. How would you do that now? I'm sorry to see it go, but I think there are no RecursionSafeTimers in the compiler anymore.

Reply:
I think you have to use the full `-stats-dir` output.

Reply:
Yeah, that's a loss, but not your fault, since the flag doesn't work anymore.